predict.py

```diff
@@ -15,28 +15,13 @@
 from abeja.datasets import Client as DatasetsClient
 import train
 import parameters
+from utils import create_colormap
 
 
 def bitget(number, pos):
     return (number >> pos) & 1
 
 
-def create_colormap(max_num):
```

Comment:
This colormap is not related to the one of training dataset. I removed and used and alternative.